schema.sql

```diff
@@ -0,0 +1,16 @@
+CREATE DATABASE vet_clinic
+    WITH 
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    CONNECTION LIMIT = -1;
+
+CREATE TABLE IF NOT EXISTS public.animals(
+	Id SERIAL PRIMARY KEY,
+	name VARCHAR,
+	date_of_birth DATE,
+	escape_attempts INT DEFAULT 0,
+	neutered BOOLEAN,
+    weight_kg DECIMAL
+);
```

Comment:
- [ ] **[REQUIRED]** What do you think about validating the length of character needed for the name column? Let's give the `VARCHAR` a maximum length of character a user can add for the `name` column. Hint; `name VARCHAR(100)`

- [ ] **[OPTIONAL]**  Good job so far, good use of `SERIAL` which is an alias for BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE values. However, be careful when creating a reference to a SERIAL PK, since that reference column has to be of this exact type. An alternative to `Id SERIAL PRIMARY KEY` can be `id INT GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY`